docs/setup/independent/high-availability.md

```diff
@@ -394,6 +402,82 @@ The next step is to create a Load Balancer that sits in front of your master nod
 You will need to ensure that the load balancer routes to **just `master0` on port 6443**. This is because kubeadm will perform health checks using the load balancer IP. Since `master0` is set up individually first, the other masters will not have running apiservers, which will result in kubeadm hanging indefinitely.
 
 If possible, use a smart load balancing algorithm like "least connections", and use health checks so unhealthy nodes can be removed from circulation. Most providers will provide these features.
+{% endcapture %}
+
+{% capture onsite %}
+In an on-site environment there may not be a physical load balancer available. Instead, keepalived can be used to setup a virtual IP pointing to a healthy master node. The configuration shown here provides an _active/passive_ setup rather than _real_ load balancing, but it can be extended for this purpose quite easily by setting up HAProxy, nginx or similar on the master nodes (not covered here). 
```

Comment:
There are so many tools to slice this I'm not even certain we want to callout tools, b/c we don't want to be in the business of picking solutions here.  

/cc @detiber

Reply:
I do believe that outlining a method that works in simple steps does help people. When I started digging into this I found the lack of simple recipe books to get started rather frustrating. 

When operating a cluster of Linux machines, _keepalived_ is the first thing coming into my mind for setting up a virtual IP (why should I bother setting up a Pacemaker cluster?). 

However I agree that explicitly mentioning that the method described here is only one out of several ways to achieve this and listing some potential alternatives (which?) could make some sense here.

Reply:
I think it is really important that we make sure to not implicitly endorse or prescribe a certain solution here.